File: 프로그래머스/0/181913. 문자열 여러 번 뒤집기/문자열 여러 번 뒤집기.js
function solution(my_string, queries) {
    for (let i of queries) {
        let [start, end] = i;
        let subString = my_string.substring(start, end + 1);
        subString = subString.split('').reverse().join('');
        my_string = my_string.substring(0, start) + subString + my_string.substring(end + 1);
    }
    return my_string;
}

File: 프로그래머스/0/181913. 문자열 여러 번 뒤집기/문자열 여러 번 뒤집기.py
def solution(my_string, queries):
    my_string_list = list(my_string)
    
    for query in queries:
        start, end = query
        sub_list = my_string_list[start : end + 1]
        sub_list.reverse()
        my_string_list[start : end + 1] = sub_list
    return "".join(my_string_list)